src/components/Header.js

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+Header.propTypes = {
+  children: PropTypes.node,
+}
+
+export default function Header({ children }) {
+  return <Headline>{children}</Headline>
+}
+
+const Headline = styled.header`
+  font-weight: bolder;
+  text-align: center;
+  padding: 10px;
+  margin: 0;
+  font-stretch: ultra-expanded;
+  font-size: 40px;
+`
```

Comment:
You can directly export the styled component (there is no big difference between `Header` and `Headline`:

```suggestion
import styled from 'styled-components/macro'

const Headline = styled.header`
  font-weight: bolder;
  text-align: center;
  padding: 10px;
  margin: 0;
  font-stretch: ultra-expanded;
  font-size: 40px;
`

export default Headline;
```